.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
+.sass-cache/
+.gitignore
+.idea/
```

Comment:
Always add newline to the end of file.

The `\n` after addition a single line allow you to have a single line in the `git diff` instead of 2.